tests/projects/models.py

```diff
@@ -202,14 +227,16 @@ class Meta:  # type: ignore
     )
 
 
-class Tag(NamedModel):
+class Tag(models.Model):
     issues: "RelatedManager[Issue]"
 
     id = models.BigAutoField(
         verbose_name="ID",
         primary_key=True,
     )
 
+    name = models.CharField(max_length=255, unique=True)
```

Comment:
Also related to #362 . Not only `unique_together` may be the case but also a single unique field.